#!/usr/bin/env python
PACKAGE = "pa1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("linear_speed", double_t, 0, "Parameter for robot linear speed", 0.4, 0.1, 1.0)
gen.add("angular_speed", double_t, 0, "Parameter for robot angular speed", 0.5, 0.1, 1.0)
gen.add("safe_distance", double_t, 0, "Parameter for robot safety distance", 1.0, 0.7, 2.0)

exit(gen.generate(PACKAGE, "pa1", "Flock"))
